accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/YamlConfigurationAdminPluginScalarConstructor.java

```diff
@@ -23,7 +24,7 @@
  * Usually this is called with 
  * <a href="https://github.com/apache/felix-dev/tree/master/configadmin-plugins/interpolation">Felix Configadmin Interpolation Plugin</a>.
  * 
- * @see <a href="https://bitbucket.org/asomov/snakeyaml/wiki/Variable%20substitution">Variable substitution</a>
+ * @see <a href="https://bitbucket.org/snakeyaml/snakeyaml/wiki/Variable%20substitution">Variable substitution</a>
```

Comment:
should extend SafeConstructor

Reply:
If I read https://snyk.io/blog/snakeyaml-unsafe-deserialization-vulnerability/ correctly, we should really be safe by just using snakeyaml-2.0 because by default the class deserialization is not active anymore and we are not using this features anywhere.